src/jsep.js

```diff
@@ -751,6 +751,9 @@ export class Jsep {
 					}
 				}
 			}
+			else if (args.length !== separator_count) {
```

Comment:
I wonder if this would work:
```suggestion
			else if (args.length !== separator_count || separator_count === 0) {
```

i.e. either use commas to separate your arguments, or spaces, but not both.

(this is just a shot in the dark, I haven't tried it or anything)

Reply:
> It looks like this would allow either comma separators or space separators but not both ...

Yup, just like I said above. 😄 

> This could be considered a breaking change if people rely on the previous behavior.

I think it's far more likely that people depend on space-separating all arguments than the inconsistent behavior that is currently the case when these are mixed. We can always make it even stricter in the future, but IMO this has a better ratio of undersirable behavior eliminated / desirable behavior eliminated.

> But if we want to think of it as being closer to javascript expression parsing (jsep), then it should probably enforce commas...??

Hasn't that ship sailed anyway with custom operators and the like?